tests/test_email_api.py

```
@@ -493,7 +549,16 @@ def test_update_email_success(self):
             "isStarred": False,
             "isUnread": False,
             "confidence": 80,
+            # Add missing required fields for EmailResponse
+            "messageId": "msg1", "threadId": "thread1",
```

Ruff (0.11.9)

Dictionary key literal `"messageId"` repeated

Remove repeated key literal `"messageId"`

(F601)